pkg/rtc/transportmanager.go

```diff
@@ -471,9 +478,11 @@ func (t *TransportManager) handleConnectionFailed(isShortLived bool) {
 		return
 	}
 
-	t.lock.RLock()
-	iceConfig := t.iceConfig
-	t.lock.RUnlock()
+	t.lock.Lock()
+	if t.isTransportReconfigured {
```

Comment:
If client failed -> restartICE (failed) -> restartICE, does this prevent the last fallback?

Reply:
nvm, it will not